src/components/MarketList/index.js

```diff
@@ -236,15 +232,26 @@ class MarketList extends Component {
           <div className="container">
             <div className="row">
               <div className="col-xs-10 col-xs-offset-1 col-sm-12 col-sm-offset-0">
-                <button
-                  type="button"
-                  onClick={this.handleCreateMarket}
-                  className="marketStats__control btn btn-default"
-                  disabled={!this.props.defaultAccount}
-                  title={this.props.defaultAccount ? '' : 'Please connect to an ethereum network to create a market'}
+                <Tooltip
+                  overlay="You need a wallet connected in order to create a market"
+                  trigger={this.props.defaultAccount ? [] : ['hover']}
                 >
-                  Create Market
-                </button>
+                  <div
+                    className={cn({
+                      'marketStats__control--container': true,
+                      disabled: !this.props.defaultAccount,
```

Comment:
Can we have a selector for this and change the name? Something like `hasWallet`

Reply:
@andre-meyer Selector for what? We have selector for current account: ```defaultAccount: getCurrentAccount(state),```
Shall i just rename the prop?

Reply:
Make a new selector that figures out if we have a wallet. Right now it should simply return false if we have no current account, then rename the prop